paper/myst.yml

```diff
@@ -1,49 +1,50 @@
 # See docs at: https://mystmd.org/guide/frontmatter
 version: 1
 project:
-  # id: ubcgif-YEAR-AUTHORS-JOURNAL
-  # title: TITLE OF PAPER
-  # description: SHORT DESCRIPTION HERE
+  id: ubcgif-2019-heagy-oldenburg-computers-geosciences
```

Comment:
This ID _should_(?) be unique, but it seems that it is failing in curvenote. Or maybe we accidentally over-wrote it somewhere else. @stevejpurves, would you mind taking a look when you have a moment?